routemaster/models/database.rb

```diff
@@ -0,0 +1,35 @@
+require 'routemaster/models'
+require 'routemaster/mixins/redis'
+require 'singleton'
+
+module Routemaster
+  module Models
+    # The (Redis) datastore
+    class Database
```

Comment:
Just a minor nitpic, but Is a "Database" class really a model object? I feel there's a few of these in the models directory already, perhaps we are taking the MVC too far?